assets/css/experience.css

```diff
@@ -444,4 +444,61 @@ ul ol{
 
 .card2 .fa{
   color: var(--text-color);
-}
\ No newline at end of file
+}
+
+/* 
+---------------------------------
+         TEXT ANIMATIONS
+----------------------------------
+*/
+.text--blocks [data-scroll='out'] .char {
+  visibility: hidden;
+}
+.text--blocks [data-scroll='in'] {
+  overflow: hidden;
+  position: relative;
+}
+.text--blocks [data-scroll='in'] .char {
+  transition-delay: 0.5s;
```

Comment:
Why repeat the same code?
Have it in the style.css and call it in all the HTML files

Increase transition delay

Reply:
Okay, I will make the changes.